src/Components/ContactPage/AddressAndLocation.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
-import "../../styles.scss";
+import "./addressAndLocation.scss";
 import restaurant from "../../assests/images/restaurant-sv.png";
 import orange from "../../assests/images/Orange.png";
 import lemonsOnTable from "../../assests/images/lemon-on-the-table.png";
 
-export default function AddressLocation() {
+export default function AddressAndLocation() {
     return (
         <div className="address-location">
             <div className="contact-section">
```

Comment:
use more generic class names for section wrappers . this is more specific ( lemon-with-orange )